src/AppBundle/Controller/RfXAnalysisController.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Analysis page of xTools.
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
+ */
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Xtools\ProjectRepository;
+use Xtools\RFA;
+
+/**
+ * Class RfXAnalysisController
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
```

Comment:
I think we can remove them. There's nothing in the phpdoc checkstyle thing that highlights them as errors is there, if they're not there? Plenty of classes seem to have no phpdoc tags at all.